### ISSUE TITLE
Put a embed struct, a panic occurs

### ISSUE THREAD

**Issue by karasawako** [guregu/dynamo#247]
I have confirmed this with v2.2.0.
If I put the following structure to DynamoDB, a panic occurs.
```
type ServerResponse struct {
    EmbeddedID int
}
type TestAddition struct {
    ServerResponse
}
type TestItem struct {
    ID int `dynamo:"id,hash" json:"id"`
    Name string `dynamo:"name,range" json:"name"`
    Addition TestAddition `dynamo:"addition,omitempty"`
}

tbl.Put(TestItem{ID: 1, Name: "test"}).Run(ctx)
```
This is because field.isZero == nil is set on line 257 of reflect.go.
Is it possible to include an embed structure in Put?

**Comment by guregu** [comment]
Looks like a bug. Thanks for the report, I'll fix it.

**Comment by guregu** [comment]
I released v2.2.1 with a fix. Thanks again!

**Comment by karasawako** [comment]
Quick fix, thanks!
